google/cloud/storage/examples/storage_grpc_samples.cc

```diff
@@ -13,25 +13,23 @@
 // limitations under the License.
 
 #include "google/cloud/storage/examples/storage_examples_common.h"
-//! [grpc-includes]
-#include "google/cloud/storage/grpc_plugin.h"
-//! [grpc-includes]
 #include "google/cloud/internal/getenv.h"
+//! [grpc-includes] [START storage_grpc_quickstart]
+#include "google/cloud/storage/grpc_plugin.h"
+#include "google/cloud/storage/options.h"
+#include "google/cloud/common_options.h"
+#include "google/cloud/options.h"
+
```

Comment:
Remove extra blank line?  Or is it important in the final rendering?

Reply:
It is important in the final rendering. Added a comment to remind us in the future.